.github/ISSUE_TEMPLATE/simple.yml

```
@@ -0,0 +1,48 @@
+name: Other issue (Simple)
+description: Did you encounter technical problems with the build system or something else not covered by other options?
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to submit an issue!
+        **For *bug reports*, *feature requests* and *documentation issues* please switch to the correct template!**
+
+        Want to ask a question instead? Please don't open an issue!
+        Head to https://ask.openrouteservice.org
+
+
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this? (required)
+      description: Please search to see if an issue already exists for the problem you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Current behavior (required)
```

Usually, this is less of a "current behavior" and more of a problem description. Let's name it as such.
```suggestion
      label: Problem description (required)
```